frontend/src/components/transactions/ActionRow.tsx

```diff
@@ -33,17 +31,8 @@ export default class extends React.Component<Props, State> {
       action,
       showDetails
     } = this.props;
-
-    let actionKind: keyof T.Action;
-    let actionArgs: T.Action;
-    if (typeof action === "string") {
-      actionKind = action;
-      actionArgs = {} as T.Action;
-    } else {
-      actionKind = Object.keys(action)[0] as keyof T.Action;
-      actionArgs = action[actionKind] as T.Action;
-    }
-
+    let actionKind = action.kind as keyof T.Action;
+    let actionArgs = action.args;
```

Comment:
No need to have a local variable anymore, just pass `action.args` directly to the component props.

Reply:
deleted